### PR TITLE
Update how to hook up with the API server

### PR DESCRIPTION
**Pull request by mihnjong**
Update how to hook up with the API server in install-1.6.md.

/release-note-none

### CODE REVIEW

File: docs/install-1.6.md [hunk 1]
```diff
@@ -146,28 +146,37 @@ start over, check out the "Final Cleanup" section in the
...
 get the IP address that points to the `Service` that sits in front of the API server pod(s).
 If you installed the catalog with one of the `helm install` commands above, then this service 
 will be called `catalog-catalog-apiserver`, and be in the `catalog` namespace. 
 
-### Notes on Getting the IP Address
+## Getting the IP Address of the API server
 
 How you get this IP address is highly dependent on your Kubernetes installation
-method. Regardless of how you do it, do not use the Cluster IP of the 
+method. Regardless of how you do it, do not use the `CLUSTER-IP` of the
```
Comment (MHBauer) [inline docs/install-1.6.md]: Why ALL CAPS for this?

Comment (mihnjong) [reply docs/install-1.6.md]: `kubectl get svc` shows 'CLUSTER-IP'.  And, i used 'EXTERNAL-IP' below as well. Do you prefer 'Cluster IP' and 'External IP'?

Comment (MHBauer) [reply docs/install-1.6.md]: Ah, maybe we could insert a copy of `kubectl get svc` then. It does make sense in context.

Comment (mihnjong) [reply docs/install-1.6.md]: Updated. Please take another look. Thanks!

**Comment by mihnjong** [comment]
Friendly ping

File: docs/install-1.6.md [hunk 2]
```diff
@@ -146,28 +146,38 @@ start over, check out the "Final Cleanup" section in the
...
 
-### Notes on Getting the IP Address
+## Getting the IP Address of the API server
 
 How you get this IP address is highly dependent on your Kubernetes installation
-method. Regardless of how you do it, do not use the Cluster IP of the 
-`Service`. The `Service` is created as a `NodePort` in this walkthrough, you 
-will need to use the address of one of the nodes in your cluster.
+method. Regardless of how you do it, do not use the `CLUSTER-IP` from
+`kubectl get service catalog-catalog-apiserver --namespace catalog` output.
```
Comment (MHBauer) [inline docs/install-1.6.md]: This clarification helps me.

**Comment by kibbles-n-bytes** [comment]
Closing, as the aggregation is our defacto story now and we probably shouldn't recommend making a special context anymore.

**Pull request closed without merging**